.circleci/config.yml

```diff
@@ -179,7 +179,42 @@ jobs:
           channel: << pipeline.parameters.alerts-slack-channel >>
           template: basic_fail_1
 
-  e2e_environment_test:
+  e2e_environment_test_on_pr:
```

Comment:
just to be clear on what this means:  does this mean that the playwright specs in the PR are run against:

1.  the currently deployed version of the UI (the tip of the`main` branch) or
2. against a deployment of the UI code in the PR (feature branch)

I think it's 1? And that what we want is 2?